docs/CONTRIBUTING.md

```diff
@@ -97,16 +106,17 @@ docker compose exec -it client /bin/sh
 ##### Rust development with docker
 
 Sometimes it's useful to test your changes in a local docker, however the
-`docker-compose.yml` file at the root directory requires rebuilding the
-images each time you want to test the change.
+`docker-compose.yml` file at the root directory requires rebuilding the images
+each time you want to test the change.
 
 To solve this, you can use the `rust/docker-compose-dev.yml` file like
 `docker compose -f docker-compose.yml -f rust/docker-compose-dev.yml <command>`
 
 This will use locally compiled binaries situated at
 `rust/target/x86_64-unknown-musl/debug`
 
-You can also set use the env variable `COMPOSE_FILE` as explained [here](https://docs.docker.com/compose/environment-variables/envvars/#compose_file)
+You can also set use the env variable `COMPOSE_FILE` as explained
+[here](https://docs.docker.com/compose/environment-variables/envvars/#compose_file)
```

Comment:
```suggestion
You can also [set the env variable `COMPOSE_FILE`](https://docs.docker.com/compose/environment-variables/envvars/#compose_file)
```
I hear from web people now and then that links saying "here" are bad for usability and accessibility, since the link requires context and doesn't have self-contained meaning.